classes/Tools.php

```diff
@@ -488,7 +488,7 @@ public static function urlBelongsToShop($url)
     {
         $urlHost = Tools::extractHost($url);
 
-        return empty($urlHost) || $urlHost === Tools::getServerName();
+        return empty($urlHost) || ($urlHost === Tools::getServerName() || $urlHost === 'www.'.Tools::getServerName());
```

Comment:
Strange 🤔 why the `www` does not work? Does it work with another subdomain ? like aaa.myshop.com ?

Reply:
Tools::getServerName uses $_SERVER and it doesnt return the www. part for some reason